### ISSUE TITLE
Access to inline template objects results in "no-access-missing-member" rule being triggerred

### ISSUE THREAD

**Issue by DethAriel** [mgechev/codelyzer#231]
Consider this code:
```ts
@Component({
  template: `
    <div #myModal="my-modal" myModal></div>
    <button (click)="myModal.show()">Show</button>
  `,
}) export class MyComponent { }

@Directive({
  selector: '[myModal]',
  exportAs: 'my-modal',
}) export class MyModalDirective { 
  public show() { /* ... */ }
}
```

If you run `tslint` with codelyzer's `"no-access-missing-member"` rule enabled, you get an error:

> The method "myModal" that you're trying to access does not exist in the class declaration.

This is not right. Of course, `"myModal"` is indeed not a method, and it does not exist in the class declaration. Yet, this is perfectly valid code

**Comment by mgechev** [comment]
This issue will be fixed when we start gathering metadata with [ngast](https://github.com/mgechev/ngast). The timeline is not very clear but probably will happen somewhen next month. Until then, you can update your `.codelyzer.js` and add the directive declaration manually.

**Comment by Matmo10** [comment]
@mgechev Is there a way to disable this rule in an external template only, but not for the whole project? I don't have a .codelyzer.js config, I just reference the codelyzer rules from tslint.json.

**Comment by mgechev** [comment]
You can create a `.codelyzer.js` in the root of your project.

> Nevermind, all you need to do is add /* tslint:disable no-access-missing-member */ in the corresponding typescript file. :)

That's correct.

**Comment by mgechev** [comment]
Implemented by https://github.com/mgechev/codelyzer/commit/d10e9809531b6795a6ab403f87b5f68b05c8afb6.

Will be released as part of 3.0.0-beta.1.
